### ISSUE TITLE
Remove all rounded Corners or remove  firefox pointy corners

### ISSUE THREAD

**Issue by jja88** [vinceliuice/Orchis-theme#184]
Hello, i wanted to know if its possible to modify theme to remove all rounded corners( im a noob idk how)
or remove these firefox corners 
![image](https://user-images.githubusercontent.com/51754069/155879484-0c9dbf61-3b53-4a5c-962f-6ff560218e47.png)

**Comment by vinceliuice** [comment]
Nope! this is a firefox issue, only firefox team fix it or we can do nothing about it

**Comment by jja88** [comment]
> Nope! this is a firefox issue, only firefox team fix it or we can do nothing about it
I have no idea how themes work but i saw that firefox fixed the issue 
https://www.omgubuntu.co.uk/2022/02/whats-new-in-mozilla-firefox-97
i changed the theme to yaru and the corners were gone,  thought the info might help if the fix is possibe 
thank you great theme :+1:
